app/services/bubble.service.js

```diff
@@ -21,8 +23,9 @@ async function get(id) {
   }
 }
 
-async function getAll() {
-  const bubbles = await Bubble.find({});
+async function getAll({ userId }) {
+
+  const bubbles = await Bubble.find({ "users.userId": mongoose.mongo.ObjectId(userId) });
```

Comment:
Só troca esse mongo pra Types dps
```suggestion
  const bubbles = await Bubble.find({ "users.userId": mongoose.Types.ObjectId(userId) });
```